updated to include recent questions (#695)
graphprotocol/docs: website/pages/en/sunrise.mdx

### How can I ensure high quality of service and redundancy for subgraphs on The Graph Network?

> All subgraphs will be supported by the upgrade Indexer. For a higher quality of service and more robust redundancy, you can add a curation signal to subgraphs eligible for indexing rewards. At the time of writing, it is recommended that you curate your subgraph with at least 3000 GRT (per subgraph) to attract about 3 Indexers.[Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).

Author: ```suggestion
All subgraphs will be supported by the upgrade Indexer. For a higher quality of service and more robust redundancy, you can add a curation signal to subgraphs eligible for indexing rewards. At the time of writing, it is recommended that you curate your subgraph with at least 3000 GRT (per subgraph) to attract about 3 Indexers. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```

> All subgraphs will be supported by the upgrade Indexer. For a higher quality of service and more robust redundancy, you can add a curation signal to subgraphs eligible for indexing rewards. At the time of writing, it is recommended that you curate your subgraph with at least 3,000 GRT (per subgraph) to attract about 3 Indexers.[Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).

Reviewer: ```suggestion
All subgraphs will be supported by the upgrade Indexer. For a higher quality of service and more robust redundancy, you can add a curation signal to subgraphs eligible for indexing rewards. At the time of writing, it is recommended that you curate your subgraph with at least 3,000 GRT (per subgraph) to attract about 3 Indexers. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```
(missing a space)

> All subgraphs will be supported by the upgrade Indexer. For a higher quality of service and more robust redundancy, you can add a curation signal to subgraphs eligible for indexing rewards. At the time of writing, it is recommended that you curate your subgraph with at least 3,000 GRT (per subgraph) to attract about 3 Indexers.[Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).

Author: It was in the original copy written before I edited, but if it's not needed, I can remove it.

> All subgraphs will be supported by the upgrade Indexer. For a higher quality of service and more robust redundancy, you can add a curation signal to subgraphs eligible for indexing rewards. At the time of writing, it is recommended that you curate your subgraph with at least 3,000 GRT (per subgraph) to attract about 3 Indexers. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).

Author: ```suggestion
All subgraphs will be supported by the upgrade Indexer. For a higher quality of service and more robust redundancy, you can add a curation signal to subgraphs eligible for indexing rewards. At the time of writing, it is recommended that you curate your subgraph with at least 3000 GRT (per subgraph) to attract about 3 Indexers.[Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```

### What happens if I don't upgrade my subgraph?

Subgraphs will be queryable on the hosted service until June 12th. After this date, the hosted service homepage will still be accessible, however, query endpoints will no longer be available. Owners of hosted service subgraphs will still be able to upgrade their subgraphs to The Graph Network after June 12th, though earlier upgrades are entitled to [earn rewards](https://thegraph.com/sunrise-upgrade-program/). Developers will also be able to claim [auto-upgraded subgraphs](https://thegraph.com/blog/unveiling-updated-sunrise-decentralized-data/#phase-2-sunbeam).

> ### What should I do with my old subgraphs? Will they stop working and should I delete them?

Author: ```suggestion
### What should I do with my subgraphs on the hosted service? Will they stop working and should I delete them?
```

> It is not possible to delete subgraphs, but they can be deprecated. To learn how to deprecate a subgraph, check out [deprecating a subgraphs](https://thegraph.com/docs/en/managing/deprecating-a-subgraph/). Query endpoints will no longer be available, and owners will not be able to deploy new subgraphs to the hosted service.

Author: ```suggestion
It is not possible to delete subgraphs. Query endpoints will remain active until June 12th, regardless of whether they have been upgraded or not. 
```

### Will Subgraph Studio be impacted by this upgrade?

No, Subgraph Studio will not be impacted by Sunrise.

> ### Will the hosted service be closed?

Reviewer 2: Not sure what "closed" means here - maybe "What will happen to the hosted service?"

Author: ```suggestion
### What will happen to the hosted service?
```

> Yes. Query endpoints will no longer be available, and owners will not be able to deploy or query the hosted service.

Reviewer 2: ```suggestion
As of the cutoff date (12th June), endpoints will no longer be available, and owners will not be able to deploy or query the hosted service, but the hosted service UI will still show subgraph pages, and subgraph owners will be able to upgrade their subgraphs if they haven't already. The hosted service UI will be retired at a later date.
```

Author: ```suggestion
After June 12th, query endpoints will no longer be available, and owners won't be able to deploy or query the hosted service. However, the hosted service UI will still show subgraph pages, and subgraph owners will be able to upgrade their subgraphs if they haven't already. The hosted service UI will be retired at a later date. 
```

### Will subgraphs need to be re-indexed again?

> No, rest assured that your subgraph will not need to be re-indexed when it is upgraded from the hosted service to Subgraph Studio. To get additional indexing support, publish your subgraph to The Graph Network, where new Indexers will sync your subgraph. In the meantime, the upgrade Indexer will seamlessly support your existing subgraphs.

Reviewer: > To get additional indexing support, publish your subgraph to The Graph Network

Wait, is this actually needed? Doesn't the upgrade process already do this? I thought the only action needed to get additional indexing support was to curate. But this might be a huge misunderstanding on my part. cc @azf20

> No, rest assured that your subgraph will not need to be re-indexed when it is upgraded from the hosted service to Subgraph Studio. To get additional indexing support, publish your subgraph to The Graph Network, where new Indexers will sync your subgraph. In the meantime, the upgrade Indexer will seamlessly support your existing subgraphs.

Reviewer 2: This isn't quite right (subgraphs are upgraded to the network, not to the studio). Maybe something like...

No, rest assured that your subgraph will not need to be re-indexed when it is upgraded to The Graph Network. Subgraphs will be immediately available for querying, powered by the upgrade indexer, which uses the same infrastructure as the hosted service. If your subgraph is indexing a network which is eligible for indexing rewards, you can add signal to your subgraph to attract additional indexers (linking to place about adding signal)

Author: ```suggestion
No, rest assured that your subgraph will not need to be re-indexed when it is upgraded to The Graph Network. Subgraphs will be immediately available for querying, powered by the upgrade Indexer, which uses the same infrastructure as the hosted service. If your subgraph is indexing a network that is eligible for indexing rewards, you can add signal to attract indexers. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```

> ### Why do I publish my subgraph to Arbitrum?

Reviewer 2: Maybe: Why is my subgraph being published to Arbitrum, it is indexing a different network?

Author: ```suggestion
### Why is my subgraph being published to Arbitrum, is it indexing a different network?
```

> The Graph protocol originally lived on Ethereum but moved to Arbitrum to reduce gas costs for all users. Subgraphs are published to The Graph on Arbitrum so that they can be indexed by Indexers engaging with smart contracts on that chain. Arbitrum is the network that subgraphs are deployed to, but subgraphs can index any of the [supported networks](https://thegraph.com/docs/en/developing/supported-networks/)

Reviewer: ```suggestion
The Graph Network originally lived on Ethereum but moved to Arbitrum to reduce gas costs for all users. Subgraphs are published to The Graph on Arbitrum so that they can be indexed by Indexers engaging with smart contracts on that chain. Arbitrum is the network that subgraphs are deployed to, but subgraphs can index any of the [supported networks](/developing/supported-networks/).
```

> The Graph protocol originally lived on Ethereum but moved to Arbitrum to reduce gas costs for all users. Subgraphs are published to The Graph on Arbitrum so that they can be indexed by Indexers engaging with smart contracts on that chain. Arbitrum is the network that subgraphs are deployed to, but subgraphs can index any of the [supported networks](https://thegraph.com/docs/en/developing/supported-networks/)

Reviewer 2: ```suggestion
The Graph Network was originally deployed on mainnet Ethereum but moved to Arbitrum One to reduce gas costs for all users. As such any new subgraphs are published to The Graph Network on Arbitrum so that they can be supported by Indexers. Arbitrum is the network that subgraphs are published to, but subgraphs can index any of the [supported networks](https://thegraph.com/docs/en/developing/supported-networks/)
```
